src/main/scala/main/Bot.scala

```diff
@@ -14,28 +14,41 @@ import slack.rtm.SlackRtmClient
 
 import scala.collection.mutable
 import scala.concurrent.duration.FiniteDuration
-import scala.concurrent.{ExecutionContextExecutor, Future}
+import scala.concurrent.{Await, ExecutionContextExecutor, Future}
 
 object Bot {
-  val dmChannels: mutable.MutableList[String] = mutable.MutableList[String]()
-  val accessToken                             = sys.env("BOT_GITHUB_TOKEN")
-  val techLeads                               = List("padge", "unrolled", "raulchedrese")
-  var prevMessages: Map[String, String]       = Map()
-  implicit val system: ActorSystem            = ActorSystem("slack")
-  implicit val ec: ExecutionContextExecutor   = system.dispatcher
+  val dmChannels: mutable.MutableList[String]   = mutable.MutableList[String]()
+  val accessToken                               = sys.env("BOT_GITHUB_TOKEN")
+  val slackToken                                = sys.env("BOT_SLACK_TOKEN")
+  val techLeads                                 = List("padge", "unrolled", "raulchedrese")
+  val channelNameLabelMap: Map[String, String]  = Map("mystery_machine" -> "MysteryMachine", "triforce" -> "triforce", "synergyteam" -> "Synergy", "emoji" -> "emoji")
+  var channelIdLabelMap: Map[String, String]    = Map()
+  var prevMessages: Map[String, String]         = Map()
+  implicit val system: ActorSystem              = ActorSystem("slack")
+  val apiClient                                 = SlackApiClient(slackToken)
+  implicit val ec: ExecutionContextExecutor     = system.dispatcher
 
   def main(args: Array[String]): Unit = {
-    val slackToken = sys.env("BOT_SLACK_TOKEN")
-
     val client    = SlackRtmClient(slackToken, new FiniteDuration(30, TimeUnit.SECONDS))
-    val apiClient = SlackApiClient(slackToken)
     val selfId    = client.state.self.id
-
     client.onMessage { message =>
+      //Not sure if there is a way to limit how long the message stays.
+      client.indicateTyping(message.channel)
+      if (!channelIdLabelMap.contains(message.channel)) {
+        val chanFuture = apiClient.getChannelInfo(message.channel)
+        val chanRes = Await.result(chanFuture, new FiniteDuration(5, TimeUnit.SECONDS))
+        if (channelNameLabelMap.contains(chanRes.name)) {
```

Comment:
Can we pull out the channel map stuff into a separate PR? Maybe something like https://github.com/lambdista/config?